components/places/sql/create_shared_triggers.sql

```diff
@@ -260,6 +314,14 @@ BEGIN
         AND last_visit_date_remote = 0;
 END;
 
+-- Equivalent to a CASCADE foreign key action on `moz_bookmarks.parent`.
+CREATE TEMP TRIGGER moz_bookmarks_parent_afterdelete_trigger
```

Comment:
I don't understand why this is preferred - maybe a short comment to elaborate?

Reply:
Will do! (It's because the `insert: item without parent` check already checks if `NEW.parent` exists in `moz_bookmarks`, so a `FOREIGN KEY(parent)` definition will do extra work to validate that constraint again).